src/List/ListSubheader.js

```diff
@@ -0,0 +1,61 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('ListSubheader', (theme) => {
+  const { palette, typography } = theme;
+
+  return {
+    root: {
+      boxSizing: 'border-box',
+      lineHeight: '48px',
+      paddingLeft: 16,
+      width: '100%',
+      color: palette.text.secondary,
+      fontWeight: typography.fontWeightMedium,
+      fontSize: typography.fontSize,
```

Comment:
We have quite some issue regarding this point on the `master` branch. People might want to use different font-family on different part of their UI.